cli/tests/test_messages_from_files.py

```diff
@@ -44,6 +44,45 @@ def test_messages_from_csv(omop_resources: Path) -> None:
     expected_messages = [
         Message(
             procedure_occurrence_id=0,
+            pseudo_patient_id=None,
+            mrn="patient_identifier",
+            accession_number="123456789",
+            study_uid="1.2.3.4.5.6.7.8",
+            project_name="ms-pinpoint-test",
+            extract_generated_timestamp=datetime.datetime.fromisoformat("2023-01-01T00:01:00Z"),
+            study_date=datetime.date.fromisoformat("2022-01-01"),
+        ),
+    ]
+    assert messages == expected_messages
+
+
+def test_messages_from_csv_with_participant_id(omop_resources: Path) -> None:
+    """
+    Given a csv with a single dataset that has participant_id defined.
+    When the messages are generated from the directory
+    Then one message should be generated
+
+    """
+    # To get this test passing:
+    # - update core.db.models.Image to include pseudo_patient_id as a str,
+    #   make optional with default to None
+    # - update pixl_cli._database._add_images_to_session to pass pseudo_patient_id to the Image
+    # - update core.patient_queue.message.Message to include pseudo_patient_id
+    # - update pixl_cli._message_processing.messages_from_df to include
+    #   pseudo_patient_id in the Message
```

Comment:
We can remove these comment lines

Reply:
[bcced48](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/bcced48b33733010d072693f7e404b910204631f)